### ISSUE TITLE
Hardware Acceleration in Home Assistant 

### ISSUE THREAD

**Issue by steuerlexi** [AlexxIT/go2rtc#819]
Hello everyone,

I hope you’re all doing well. I’m currently exploring the Go2RTC Hardware Acceleration feature in Home Assistant and have some questions about it. Specifically, I’ve noticed that enabling [Hardware](https://community.home-assistant.io/c/hardware/22) doesn’t seem to provide clear advantages.

Could someone clarify whether the stream is actually rendered on the host after Hardware Acceleration is activated, or does it happen later at the client? I’m not entirely clear on how this process works and whether there are specific scenarios where Hardware Acceleration does offer noticeable benefits.

**Comment by AlexxIT** [comment]
Have you read this? https://github.com/AlexxIT/go2rtc/wiki/Hardware-acceleration

**Comment by steuerlexi** [comment]
yes, I have read it but I wasn´t smarter afterwards :-).   Does it bring advantages if i activated GPU on the ESXi Host? I am running a HAOS VM there.
